data/human/names.txt

```diff
@@ -674,6 +690,20 @@ phrase "male titles"
 		"Duke"
 		"Baron"
 		"Master"
+		"Brother"
```

Comment:
Some of these are maybe a little too religious or obscure.

Reply:
I think they're fine.